weekly/2023/week-38.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Week 38 in Packit
+date: 2023-09-25
+tags:
+  - 2023-September
+  - 2023
+  - September
+---
+
+## Week 38 (September 19th – September 25th)
+
+- Packit now supports the `pkg_tool` option in the config (at the top-level or with specific packages when using the monorepo syntax). This option can be used for switching between `fedpkg` or `centpkg`. ([packit#2085](https://github.com/packit/packit/pull/2085))
```

Comment:
maybe I would mention that this is part of the effort of implementing syncing release for CentOS Stream for the context?